built_value/pubspec.yaml

```diff
@@ -6,7 +6,7 @@ description: >
 homepage: https://github.com/google/built_value.dart
 
 environment:
-  sdk: '>=2.9.0 <2.10.0'
+  sdk: '>=2.9.0-18.0 <2.10.0'
```

Comment:
I'm guessing that will need to change soon ;)

Reply:
Likely :)

Doesn't have any lasting effect until publishing, though, and I won't publish this until there is a fixed version with null safety support. So, nothing to worry about for now :)